entities/Rules/index.js

```diff
@@ -157,30 +159,22 @@ module.exports = {
   ],
   PAY_FINE: [
     ({ UI }) => UI.payFine(),
-    (_, gameState) => {
-      // potentially entering negative wealth here, will be resolved in subsequent rule
-      // TODO: WEALTHSERVICE: Check Liquidity
-      // what happens if current player doesn't have enough?
+    ({ UI, notify }, gameState) => {
+      // TODO: add extra check if bankrupt and out of the game
+      while (gameState.currentPlayer.cash < gameState.config.fineAmount) {
```

Comment:
Won't this continuously queue `'LIQUIDATION'` events ?
Not sure if I am forgetting or missing something re: the Event Bus.. :S

Reply:
We'll be good for this. Notify is synchronous.